### PR TITLE
Update the Facebook Adapter to store a boolean state for whether it started presenting.

### PR DESCRIPTION
**Pull request by chlandacastro**
Update the Facebook Adapter to store a boolean state for whether it started presenting.

### CODE REVIEW

**Comment by ericleich** [comment]
@xilosada @yroxe FYI we also merged https://github.com/googleads/googleads-mobile-ios-mediation/commit/75b81833433701e353a778e02ef5adf41dcf4ef7 to explicitly call showAd instead of checking for isValid first. Should help with debugging.

File: adapters/Facebook/FacebookAdapter/OpenBidding/GADFBInterstitialRenderer.m [hunk 1]
```diff
@@ -96,6 +99,11 @@ - (void)interstitialAdDidLoad:(FBInterstitialAd *)interstitialAd {
 }
 
 - (void)interstitialAd:(FBInterstitialAd *)interstitialAd didFailWithError:(NSError *)error {
+  if (_presentCalled) {
+    NSLog(@"Received a Facebook SDK error during presentation: $@", error.localizedDescription);
+    _adLoadCompletionHandler(nil, error);
```
Comment (yroxe) [inline adapters/Facebook/FacebookAdapter/OpenBidding/GADFBInterstitialRenderer.m:104]: Can we drop rows 104-105 as we still send the error on 107?

File: adapters/Facebook/FacebookAdapter/OpenBidding/GADFBRewardedRenderer.m [hunk 2]
```diff
@@ -108,6 +111,11 @@ - (void)rewardedVideoAdDidLoad:(FBRewardedVideoAd *)rewardedVideoAd {
 }
 
 - (void)rewardedVideoAd:(FBRewardedVideoAd *)rewardedVideoAd didFailWithError:(NSError *)error {
+  if (_presentCalled) {
+    NSLog(@"Received a Facebook SDK error during presentation: $@", error.localizedDescription);
+    _adLoadCompletionHandler(nil, error);
```
Comment (yroxe) [inline adapters/Facebook/FacebookAdapter/OpenBidding/GADFBRewardedRenderer.m:116]: Can we drop rows 116-117 as we still send the error on 119?

File: adapters/Facebook/FacebookAdapter/OpenBidding/GADFBInterstitialRenderer.m (hunk 1 above)
Comment (chlandacastro) [reply adapters/Facebook/FacebookAdapter/OpenBidding/GADFBInterstitialRenderer.m:104]: It was a code error, and it was corrected. Line 104 should be [_adEventDelegate didFailToPresentWithError:error];

File: adapters/Facebook/FacebookAdapter/OpenBidding/GADFBRewardedRenderer.m (hunk 2 above)
Comment (chlandacastro) [reply adapters/Facebook/FacebookAdapter/OpenBidding/GADFBRewardedRenderer.m:116]: It was a code error, and it was corrected. Line 116 should be [_adEventDelegate didFailToPresentWithError:error];

**Pull request merged**